client/Cargo.toml

```diff
@@ -36,3 +36,11 @@ metrics = { path = "../common/metrics" }
 proto_conv = { path = "../common/proto_conv" }
 types = { path = "../types" }
 vm_genesis = { path = "../language/vm/vm_genesis" }
+
+[dev-dependencies]
+nextgen_crypto = { path = "../crypto/nextgen_crypto", features = ["testing"] }
+types = { path = "../types", features = ["testing"]}
+
+[features]
+default = []
```

Comment:
What do these two lines mean here?

Reply:
this is the list of configuration flags that you can use to build this module. 
By default you don't use any (`default = []`), but you can use the `testing` flag when building, which will make use of two other features defined in different crates (line below).